frontend/.eslintrc.js

```diff
@@ -67,6 +67,7 @@ module.exports = {
         "testing-library/prefer-screen-queries": "warn",
         // Prevent unnecessary console statements
         "no-console": ["error", { allow: ["warn", "error"] }],
+        "promise/catch-or-return": ["error", { allowFinally: true }],
```

Comment:
this just seems logical - otherwise we get a linting error if a `finally` call is made at the end of an unreturned promise chain